deploy/architecture.md

```diff
@@ -4,13 +4,52 @@
 
 See [introduction](../build/pooling.md#introduction)
 
-## Architecture
+## Definitions and terminology notes
 
-See [architecture](../build/deploy/README.md#architecture)
+This section defines some concepts and related references used in this documentation.
+
+### DSS Region
+
+A DSS Region is a region in which a single, unified airspace representation is
+presented by one or more interoperable DSS instances, each instance typically
+operated by a separate organization.  A specific environment (for example,
+"production" or "staging") in a particular DSS Region is called a "pool".
+
+### DSS Pool
+
+A DSS Pool is a set of interoperable and interconnected DSS instances in a specific
+DSS Region. Each instance is typically operated by a separate organization.
+
+### DSS instance
+
+A DSS instance is a single logical replica in a DSS pool hosted by a single
```

Comment:
Could this 'single logical replica' be on several CRDB nodes? I.e. can a DSS instance host several CRDB nodes?